### ISSUE TITLE
Bug: CSS icon's text has a bug with its color

### ISSUE THREAD

**Issue by sepsol** [devicons/devicon#258]
```
<svg viewBox="0 0 128 128">
  <path fill="#000" d="M19.67 26l8.069 90.493 36.206 10.05 36.307-10.063 8.078-90.48h-88.66zm69.21 50.488l-2.35 21.892.009 1.875-22.539 6.295v.001l-.018.015-22.719-6.225-1.537-17.341h11.141l.79 8.766 12.347 3.295-.004.015v-.032l12.394-3.495 1.308-14.549h-25.907000000000004l-.222-2.355-.506-5.647-.265-2.998h27.886000000000003l1.014-11h-42.473l-.223-2.589-.506-6.03-.265-3.381h55.597l-.267 3.334-2.685 30.154"></path><path fill="#1572B6" d="M89 14.374l-7.149-8.374h7.149v-5h-16v4.363l8.39 7.637h-8.39v5h16zM70 14.374l-6.807-8.374h6.807v-5h-15v4.363l7.733 7.637h-7.733v5h15zM52 13h-8v-7h8v-5h-14v17h14z"></path>
</svg>
```

![image](https://user-images.githubusercontent.com/47871887/89513758-39303080-d7ea-11ea-8e3a-6011f31f4d6e.png)

**Comment by amacado** [comment]
@sepsol please provide some more information. What do you mean with "has a bug"? I also have no idea where the code you provided is from. It's not the svg code from our repo (which is displaying the icon as expected).

```
<svg viewBox="0 0 128 128">
<path fill="#1572B6" d="M19.67 26l8.069 90.493 36.206 10.05 36.307-10.063 8.078-90.48h-88.66zm69.21 50.488l-2.35 21.892.009 1.875-22.539 6.295v.001l-.018.015-22.719-6.225-1.537-17.341h11.141l.79 8.766 12.347 3.295-.004.015v-.032l12.394-3.495 1.308-14.549h-25.907000000000004l-.222-2.355-.506-5.647-.265-2.998h27.886000000000003l1.014-11h-42.473l-.223-2.589-.506-6.03-.265-3.381h55.597l-.267 3.334-2.685 30.154"></path><path fill="#1572B6" d="M89 14.374l-7.149-8.374h7.149v-5h-16v4.363l8.39 7.637h-8.39v5h16zM70 14.374l-6.807-8.374h6.807v-5h-15v4.363l7.733 7.637h-7.733v5h15zM52 13h-8v-7h8v-5h-14v17h14z"></path>
</svg>
```

![image](https://user-images.githubusercontent.com/8781699/89515733-d03fbd00-d7d7-11ea-90c5-699bc6e6859f.png)

**Comment by amacado** [comment]
I'm closing this issue, when you can specify the bug feel free to reopen it :)

**Comment by sepsol** [comment]
I'm sorry for the delay. It was my first time working with svg files, I managed to fix my problem with it. Basically there were two or more path "d"s inside the svg tag which caused the issue.

The issue was that when I copied the code for the monocolor svg version from your website and tried to change the color of the icon, the color of the "css" text remained blue and didn't change to what I assigned to the first "d" attribute. I basically removed the middle "d"s and it worked fine. There  were some more logos that were having this issue though. I didn't list them but since I've used somewhere around 17 of your logos, I can tell you all of their names :)
